example/android/app/build.gradle

```diff
@@ -49,6 +49,7 @@ android {
 
 dependencies {
     implementation 'com.android.support:multidex:1.0.3'
+//    implementation platform('com.google.firebase:firebase-bom:31.4.0')
```

Comment:
Is it worth removing this, or should it stay commented out?

Reply:
Yeah. Stay commented out as of now.